backend/k6/scripts/get-statistic/get-statistic-run.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# InfluxDB 환경 변수 설정
+if [ -f .env ]; then
+  export $(cat .env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)